Core/HLE/sceNetAdhoc.cpp

```diff
@@ -1809,7 +1809,8 @@ static int sceNetAdhocPdpRecv(int id, void *addr, void * port, void *buf, void *
 						return WaitBlockingAdhocSocket(threadSocketId, PDP_RECV, id, buf, len, timeout, saddr, sport, "pdp recv");
 					}
 
-					return hleLogSuccessVerboseX(SCENET, ERROR_NET_ADHOC_WOULD_BLOCK, "would block");
+					VERBOSE_LOG(SCENET, "%08x=sceNetAdhocPdpRecv: would block", ERROR_NET_ADHOC_WOULD_BLOCK); // Temporary fix to avoid a crash on the Logs due to trying to Logs syscall's argument from another thread (ie. AdhocMatchingInput thread)
+					return ERROR_NET_ADHOC_WOULD_BLOCK; // hleLogSuccessVerboseX(SCENET, ERROR_NET_ADHOC_WOULD_BLOCK, "would block");
```

Comment:
I think it's kinda weird to only change one return point of the function, there are more that use hleLog. But I guess if it solves the problem for now...

Reply:
Because it's the only return code that floods the the log from AdhocMatching thread

Reply:
Doesn't mean the others are completely safe though? But I guess I'm ok with this as a temporary fix.

Reply:
Yes, but we'll be changing the thread into PSPThread in the future so the logs won't be an issue anymore.